Add information to encourage users to use Windows 11. (#3695)
dfinity/portal: docs/developer-docs/getting-started/install/index.mdx

#### Using WSL 2
Follow Microsoft&#39;s instructions for installing the <a href="https://docs.microsoft.com/en-us/windows/wsl/install">Windows Subsystem for Linux</a>. Make sure you are running Windows 10 (version 2004 or higher) or Windows 11.

:::info

> Even you can run WSL 2 on both Windows 10 and Windows 11, Windows 11 is preferred as it has better support for WSL 2.

Reviewer: ```suggestion
WSL 2 is supported on both Windows 10 and Windows 11, although Windows 11 is preferred as it has better support for WSL 2.
```
